src/scss/table-override.scss

```diff
@@ -0,0 +1,138 @@
+/*** Commented out all the style overrides **/
+$table: ();
+
+:root {
+  // --rh-color-light-gray: #fafafa;
+  // --rh-color-gray-300: #d2d2d2;
+  // --rh-color-brand-yellow: #f0ab00;
+  // --rh-color-gray-700: #6a6e73;
+}
+
+:root:where(.pf-v6-theme-dark) {
+  // --rh-color-light-gray: #fafafa;
+  // --rh-color-gray-300: #d2d2d2;
+  // --rh-color-brand-yellow: #f0ab00;
+  // --rh-color-gray-700: #6a6e73;
+}
+
+.pf-v6-c-table {
+  /* Header styles */
+  --pf-v6-c-table__thead--cell--FontWeight: var(--rh-font-weight-heading-bold, 700);
+  --pf-v6-c-table__thead--cell--FontSize: var(--rh-font-size-code-sm, 14px);
```

Comment:
I believe this should either be the heading font-size token or the regular text font-size token, but not the code font-size token.